Nos/Views/NIP05View.swift

```diff
@@ -43,13 +43,17 @@ struct NIP05View: View {
                             username: nip05Identifier,
                             publicKey: publicKey
                         )
+                    } catch URLError.cancelled {
+                        // Cancelled requests are common in bad networks. We
+                        // should keep things as they were to avoid reproducing
+                        // https://github.com/planetary-social/nos/issues/1161
```

Comment:
I appreciate the comment here to explain what's happening. I think I'd go a little further -- in order to fully understand this comment, you need to open the link. What if we include a brief description of the issue right here so the reader doesn't have to follow the link? We can still keep the link so they can get more detail.

Reply:
Yeah, I agree with you. Let me correct it.